### PR TITLE
OKTA-809675 OIN Wizard - Navigate to Test integrations page

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** And "Navigate to Test integrations page" in the oIN Wizard
- **Is this PR related to a Monolith release?** Yes, 2024.11.0

### Resolves:

* [OKTA-809675](https://oktainc.atlassian.net/browse/OKTA-809675)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/85e72eaff89d47b21617a904b3329a36f82f6b00/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4b1a40e7-ce0d-4014-a4f1-af84073fa059) | 78  | 85  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.21a7c418-570b-4c2b-ac29-b948a90cd4c9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/122bc80bd94a3eb82a7b5a96bf796e5dd0fd3ec6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0d0faaf5-c52f-4bf6-9c28-a653f8842204) | 78  | 85  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.32575998-6fd4-4009-bffa-b10362908e43?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Start your integration submission for OIN publication:
     > **Note:** Submit your integration from an Okta account that has your company domain in the email address. You can't use an account with a personal email address. The OIN team doesn't review submissions from personal email accounts.
 
 1. Go to **Applications** > **Your OIN Integrations** in the Admin Console.
+
+   > **Note:** If want to only test an existing submission, see [Navigate directly to test your integration](#navigate-directly-to-test-your-integration).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:71]:
```suggestion
   > **Note:** If you only want to test an existing submission, see [Navigate directly to test your integration](#navigate-directly-to-test-your-integration).
```

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 2]
```diff
@@ -183,6 +186,17 @@ The OIN Wizard journey includes the **Test integration** experience page to help
 
 > **Note:** You must have the Okta Browser Plugin installed with **Allow in Incognito** enabled before you use the  **OIN Submission Tester**. See [OIN Wizard requirements](/docs/guides/submit-app-prereq/main/#oin-wizard-requirements).
 
+#### Navigate directly to test your integration
+
+You can navigate directly to the OIN Wizard **Test integration** page if you have an existing submission in the **Your OIN Integrations** dashboard. This option is only available if your submission has protocol, configuration, and test information already saved. You can bypass the **[Select protocol](#start-a-submission)**, **[Configure your integration](#configure-your-integration)**, and **[Test your integration](#enter-test-information)** pages in the OIN Wizard, and start generating instances for testing. This saves you time and avoids unnecessary updates to an existing integration submission.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:191]:
```suggestion
You can navigate directly to the OIN Wizard **Test integration** page if you have an existing submission in the **Your OIN Integrations** dashboard. This option is only available if your submission has protocol, configuration, and test information. You can bypass the **[Select protocol](#start-a-submission)**, **[Configure your integration](#configure-your-integration)**, and **[Test your integration](#enter-test-information)** pages in the OIN Wizard, and start generating instances for testing. This saves you time and avoids unnecessary updates to an existing integration submission.
```

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 3]
```diff
@@ -183,6 +186,17 @@ The OIN Wizard journey includes the **Test integration** experience page to help
...
+
+To navigate directly to test your integration:
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:193]:
```suggestion
Follow these steps to test your integration:
```

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 4]
```diff
@@ -183,6 +186,17 @@ The OIN Wizard journey includes the **Test integration** experience page to help
...
+
+1. From the **Your OIN Integrations** dashboard, select the more menu icon (![three-dot more icon](/img/icons/odyssey/more.svg)) next to the integration submission you want to test.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:195]: Should more be capitalized and in bold?

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 5]
```diff
@@ -183,6 +186,17 @@ The OIN Wizard journey includes the **Test integration** experience page to help
...
+1. Select **Test your integration** from the more menu.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:196]: Should more be capitalized and in bold?

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/720c022b645783b6eb2c317ee8efe1d7b0931f42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4df85a4b-8ba8-4ff4-87ea-147506c1757e) | 78  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.95c1a7ef-6033-4967-a33a-918786674789?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:193]: This is a shortcut to "test your integration". Maybe I need to rework this part. Thanks

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md (hunk 4 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:195]: "more" isn't in the UI. The icon is the three-dot "action menu". When I asked the infodev-standards channel earlier for this, they recommended that I use "more" to describe the icon. I'm open to using another term for this icon.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/4350c0a86b2d5f201d7e161268630b99a16fda7c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/21c33772-6336-4e97-a407-e966f491c8d6) | 78  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4c803db5-8de4-4ab9-b434-95f06060dc70?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/879ccfd3b9609b18602279eb146cd7b83486b689/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d082d161-6780-4aa9-be90-99738acc0b14) | 78  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.18b42817-aa48-49b6-8590-3314b02ce42d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 6]
```diff
@@ -183,6 +186,19 @@ The OIN Wizard journey includes the **Test integration** experience page to help
 
 > **Note:** You must have the Okta Browser Plugin installed with **Allow in Incognito** enabled before you use the  **OIN Submission Tester**. See [OIN Wizard requirements](/docs/guides/submit-app-prereq/main/#oin-wizard-requirements).
 
+#### Navigate directly to test your integration
+
+You can navigate directly to the OIN Wizard **Test integration** page if you have an existing submission in the **Your OIN Integrations** dashboard. You can bypass the **[Select protocol](#start-a-submission)**, **[Configure your integration](#configure-your-integration)**, and **[Test your integration](#enter-test-information)** pages in the OIN Wizard, and start generating instances for testing. This saves you time and avoids unnecessary updates to an existing integration submission.
+
+Follow these steps to bypass the configuration pages in the OIN Wizard:
+
+1. From the **Your OIN Integrations** dashboard, select the more menu icon (![three-dot more icon](/img/icons/odyssey/more.svg)) next to the integration submission you want to test.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:195]:
```suggestion
1. From the **Your OIN Integrations** dashboard, select the more icon (![three-dot more icon](/img/icons/odyssey/more.svg)) next to the integration submission that you want to test.
```

File: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md [hunk 7]
```diff
@@ -183,6 +186,19 @@ The OIN Wizard journey includes the **Test integration** experience page to help
...
+1. Select **Test your integration** from the menu.
+
+   The OIN Wizard **Test integration** page appears for you to generate an instance and test your integration.
+
+    > **Note:** If you haven't specified test information in the **[Test your integration](#enter-test-information)** page, then you are directed to this page to enter testing details. You can go to the **Test integration** page only if the protocols, configuration, and test details are provided in your submission.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md:200]:
```suggestion
    > **Note:** If you haven't specified test information in the **[Test your integration](#enter-test-information)** page, then you're directed to this page to enter testing details. You can go to the **Test integration** page only if the protocols, configuration, and test details are provided in your submission.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/571ada45746c99aff6e2da0035c15f572cd47e36/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5bbc2171-cdc8-4c31-b39f-3c61655eb8cf) | 78  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.aacb6adf-de1f-416f-a51f-28f994659ea7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/97d29ae8da88bf113246c89b13d0568430b7936d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a05ee21c-d4cd-4da0-a64c-5855aa097545) | 78  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a67295f4-7a8d-4681-98b5-08ce86772209?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**